### PR TITLE
[15.0][IMP] l10n_es: Establish more appropriate cash_difference accounts

### PR DESCRIPTION
**Pull request by carlosdauden**
Description of the issue/feature this PR addresses:

Current behavior before PR:
Accounts 999001 and 999002 for default_cash_difference_income_account_id and default_cash_difference_expense_account_id are created instead of use 778000 and 678000

5729991 account is created with length 7 instado of 6

Desired behavior after PR is merged:
Use 431200 and 411000
Use 778000 and 678000
Create 572991 account with length 6

@Tecnativa

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_es/data/account_chart_template_account_account_link.xml [hunk 1]
```diff
@@ -15,6 +15,9 @@
             <field name="default_cash_difference_income_account_id" ref="account_common_778"/>
             <field name="default_cash_difference_expense_account_id" ref="account_common_678"/>
             <field name="default_pos_receivable_account_id" ref="account_common_4301" />
+            <field name="account_journal_suspense_account_id" ref="account_common_572991" />
+            <field name="account_journal_payment_debit_account_id" ref="account_common_572992" />
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_chart_template_account_account_link.xml:19]: This one should be 4312, changing the type to current assets.

File: addons/l10n_es/data/account_chart_template_account_account_link.xml [hunk 2]
```diff
@@ -15,6 +15,9 @@
...
+            <field name="account_journal_payment_credit_account_id" ref="account_common_572993" />
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account_chart_template_account_account_link.xml:20]: And this one 4110, changing also the type.

File: addons/l10n_es/data/account_chart_template_account_account_link.xml (hunk 1 above)
Comment (carlosdauden) [reply addons/l10n_es/data/account_chart_template_account_account_link.xml]: Of course

File: addons/l10n_es/data/account_chart_template_account_account_link.xml (hunk 2 above)
Comment (carlosdauden) [reply addons/l10n_es/data/account_chart_template_account_account_link.xml]: Of course

File: addons/l10n_es/data/account.account.template-common.csv [hunk 3]
```diff
@@ -340,6 +340,7 @@
 "account_common_570","Caja, euros","570","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
 "account_common_571","Caja, moneda extranjera","571","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
 "pgc_572_child","Bancos e instituciones de crédito c/c vista, euros","572","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
+"account_common_572998","Cuenta transitoria","572998","account.data_account_type_current_assets","l10n_es.account_chart_template_common","False"
```
Comment (pedrobaeza) [inline addons/l10n_es/data/account.account.template-common.csv:343]: Why adding this one manually?

Comment (pedrobaeza) [reply addons/l10n_es/data/account.account.template-common.csv:343]: I mean, add `572999`

**Comment by oco-odoo** [comment]
@carlosdauden All in all, I have no problem with the diff, but could you answer @pedrobaeza 's comment ?

File: addons/l10n_es/data/account.account.template-common.csv (hunk 3 above)
Comment (carlosdauden) [reply addons/l10n_es/data/account.account.template-common.csv:343]: When I use 572999 this account is set as Internal Transfer Account too:
image

If I use 572998 a new account is created from transfer_account_code_prefix and it is used as Internal Transfer Account (expected behavior):
image

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
OK then. @oco-odoo the explanation seems reasonable for me.

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Comment by carlosdauden** [comment]
squashed and rebased

**Comment by oco-odoo** [comment]
@carlosdauden Your commit message is very unclear. The description and title of your PR are ok, though. I suggest reusing them ;)

**Comment by carlosdauden** [comment]
Commit message changed

**Comment by oco-odoo** [comment]
@robodoo r+

**Review by rafaelbn — COMMENTED** [review]
Thank you! 👍

**Pull request closed without merging**